templates/zerver/help/make-a-user-an-administrator.md

```diff
@@ -0,0 +1,29 @@
+# Make a user an administrator
+
+> With great power comes great responsibility.
+
+By default, everyone in a realm is a user which limits them from modifying realm wide settings, such as the organization name, activating/deactivating users, deleting streams, etc. Zulip realm administrators can give a user administrative rights. To do so, follow the steps below:
```

Comment:
such as changing the organization name, activating/deactivating users, deleting streams, etc.